content/get-haiku/r1beta2/release-notes.md

```diff
@@ -0,0 +1,236 @@
+
+### Improved HiDPI support
+
+Begun but significantly incomplete at the time of Beta 1, this release sees a significant advancement in support for HiDPI monitors (and UI scaling in general.) Unlike other operating systems which largely enact a "device pixel ratio", that converts a number of "internal pixels" to "screen pixels", Haiku instead scales everything based on font size, and leaves pixels to mean "screen pixels" in isolation. This also means that one can effectively change the UI size and scale on any display by changing the system font size, and everything else should adjust to match it.
+
+Below are two screenshots, showing ActivityMonitor and AboutSystem with a 12pt font size on the left (Haiku's default), and with an 18pt font size on the right (in other words, 150% of "normal.")
+
+<style>td>img{max-width:400px;}</style>
+<table><tr><td><img src="/files/get-haiku/r1beta2/normal-activitymonitor-aboutsystem.png"></td><td><img src="/files/get-haiku/r1beta2/hidpi-activitymonitor-aboutsystem.jpg"></td></tr></table>
+
+There are still some bugs to be ironed out (such as scrollbars not drawing with proper scaling applied, as seen in this screenshot), but things are now generally usable under HiDPI. Other applications, such as Deskbar and Tracker, saw significant improvements to their behavior on HiDPI, as well.
+
+### Deskbar improvements
+
+The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle to the top of the screen.
+
+<img src="/files/get-haiku/r1beta2/deskbar_minimode.png"/><br>
+<small><i>New mini-mode to put the Deskbar at the top of the screen.</i></small>
+
+Furthermore, there are more options for showing larger icons, which is especially useful on screens with a higher DPI.
+
+<img src="/files/get-haiku/r1beta2/deskbar_icon_sizes.png"/><br>
+<small><i>A new slider to change the icon size in Deskbar</i></small>
+
+### New Input preferences
+
+The Mouse, Keyboard, Touchpad and Joystick preferences have been consolidated into one Input preferences panel.
+
+<img src="/files/get-haiku/r1beta2/input_preferences.png"/><br>
+<small><i>The consolidated Input preferences, including support for mice with 5 buttons.</i></small>
+
+Additionally, there is now support for mice with more than three buttons, including support for button remapping.
+
+### WebPositive improvements
+
+Haiku's built-in web browser, WebPositive, now uses a newer version of WebKit. Additionally, it contains a number of fixes to the Haiku backend, most notably to reserve significantly less memory, and to fix WebSockets, as well as a variety of crashes.
+
+### More ported software
+
+Thanks to Haiku's relatively robust POSIX compliance, as well as ports of open source GUI toolkits, even more pieces of popular open source software have been ported to Haiku since Beta 1 and are available in the package repositories.
```

Comment:
Screenshots:
1. Ported applications:
![PortedApps](https://user-images.githubusercontent.com/2559260/84137217-b6ac1e80-aa8f-11ea-8361-2ea9bcb7f8a4.png)

2. Games
![Games](https://user-images.githubusercontent.com/2559260/84137387-fa9f2380-aa8f-11ea-9b8a-5d161231b803.png)

3. Games (opensource and free only)
![FreeGamesOnly](https://user-images.githubusercontent.com/2559260/84137568-3df99200-aa90-11ea-9c5c-dba8df1272e0.png)

Reply:
These are great, thanks! I'll use the first and last of these.

Reply:
Something on the last too many penguins ;) Maybe make a new one with free & opensource games where there will be fewer penguins?

Reply:
I think non-free games are fine too? Why not?

Reply:
New screnshot without tuxes )
![screenshot72](https://user-images.githubusercontent.com/2559260/84159360-5679a480-aab0-11ea-8c4f-f10d499408c3.png)